### PR TITLE
support(ci): fix conditional binary operator expected

### PR DESCRIPTION
**Pull request by Justkant**
### 📝 Description

Fix a bad usage of `if` with a single command, 2 options here:
```bash
if git merge release
then
```
or
```bash
if [[ $(git merge release) ]]
then
```

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .github/workflows/release-prepare-hotfix.yml [hunk 1]
```diff
@@ -85,9 +85,9 @@ jobs:
       - name: merge into release if present
         id: merge-release
         run: |
-          if [[ git checkout release ]]
+          if git checkout release
           then
-            if [[ git merge ${{ github.event.inputs.ref }} ]]
+            if git merge ${{ github.event.inputs.ref }}
```
Comment (elbywan) [inline .github/workflows/release-prepare-hotfix.yml:90]: if the merge fails, wondering if `git merge --abort` would be needed?

Comment (Justkant) [reply .github/workflows/release-prepare-hotfix.yml:90]: Good point, could be good to do at least to be safe

Comment (Justkant) [reply .github/workflows/release-prepare-hotfix.yml:91]: Added

**Review by elbywan — APPROVED** [review]
LGTM 👌

**Pull request merged**